sonar-python-plugin/src/test/java/org/sonar/plugins/python/PythonSquidSensorTest.java

```diff
@@ -73,15 +73,16 @@ public void should_execute_on_python_project() {
     Project project = mock(Project.class);
     assertThat(sensor.toString()).isEqualTo("PythonSquidSensor");
     assertThat(sensor.shouldExecuteOnProject(project)).isFalse();
-    fs.add(new DefaultInputFile("test.py").setLanguage(Python.KEY));
+    fs.add(new DefaultInputFile(TestUtils.MODULE, "test.py").setLanguage(Python.KEY));
     assertThat(sensor.shouldExecuteOnProject(project)).isTrue();
   }
 
   @Test
   public void should_analyse() {
     String relativePath = "src/test/resources/org/sonar/plugins/python/code_chunks_2.py";
-    DefaultInputFile inputFile = new DefaultInputFile(relativePath).setLanguage(Python.KEY);
-    inputFile.setAbsolutePath((new File(relativePath)).getAbsolutePath());
+    DefaultInputFile inputFile = new DefaultInputFile(TestUtils.MODULE, relativePath).setLanguage(Python.KEY);
+    
+    inputFile.setModuleBaseDir(Paths.get("").toAbsolutePath());
```

Comment:
@ivandalbosco We can refactor `Paths.get("")` into this class' private static field